src/containers/Secrets/Secrets.scss

```diff
@@ -0,0 +1,4 @@
+.bx--tooltip__label {
+  font-size: 120% !important;
```

Comment:
Maybe the text shouldn't be part of the tooltip? Also let's try to avoid `!important` as much as possible, instead we can make the selector slightly more specific (add a class to a parent element that we can reference) so we're only targeting this instance and it won't be overridden by more general styles.

Reply:
Hmm, what do you mean by not being part of the tooltip? 

Just some text above the type perhaps instead of using a tooltip? Or, having something like <Tooltip><our own div with ID that we style>...</div></Tooltip>?

Reply:
Actually it does look a bit weird on the form if this has its own style so I'm gonna omit it